.yarnrc.yml

```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```

Comment:
Can you remove this file: .yarnrc.yml?  It seems to cause unexpected changes to the yarn.lock file

Reply:
okay.